tests/starlark/eval-fn-config/Kptfile

```diff
@@ -0,0 +1,8 @@
+apiVersion: kpt.dev/v1
+kind: Kptfile
+metadata:
+  name: example
+pipeline:
+  mutators:
+    - image: gcr.io/kpt-fn/starlark:unstable
```

Comment:
We should also have a test case where the functionConfig is not referenced in the pipeline (that's the scenario use ran into) because then it it's part of meta resources.

Reply:
Done